cirq/google/json_test_data/spec.py

```diff
@@ -7,7 +7,7 @@
 
 TestSpec = ModuleJsonTestSpec(
     name="cirq.google",
-    packages=[cirq.google],
+    packages=[cirq.google, cirq.google.experimental],
```

Comment:
Should whether `cirq.google` passes CI rely on experimental objects passing JSON serialization? (Not opposed, just making sure)

Reply:
Good question, I have a slight preference for yes, make the decisions earlier rather than later. This ensures that when we "graduate" a module to the parent package, we wouldn't have to do a massive json rework.